### PR TITLE
Release/v0.22.2

### PR DESCRIPTION
**Pull request by ebuchman**
* [ ] Updated all relevant documentation in docs
* [ ] Updated all code comments where relevant
* [ ] Wrote tests
* [ ] Updated CHANGELOG.md

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -88,7 +88,7 @@
 
 [[constraint]]
   name = "github.com/prometheus/client_golang"
-  version = "0.8.0"
+  branch = "master"
```
Comment (ebuchman) [inline Gopkg.toml]: we shouldnt do this

File: docs/interviews/tendermint-bft.md [hunk 2]
```diff
@@ -0,0 +1,250 @@
+# Interview Transcript with Tendermint core researcher, Zarko Milosevic, by Chjango
```
Comment (ebuchman) [inline docs/interviews/tendermint-bft.md]: weird that this lives in here?

File: Gopkg.toml (hunk 1 above)
Comment (melekes) [reply Gopkg.toml]: revision?

File: docs/interviews/tendermint-bft.md (hunk 2 above)
Comment (greg-szabo) [reply docs/interviews/tendermint-bft.md]: lol

Comment (melekes) [reply docs/interviews/tendermint-bft.md]: yep, if this is published, we can remove it

**Pull request merged**